### PR TITLE
Added blurb about extensions with link to NTIAs and hopefully others

### PR DESCRIPTION
**Pull request by 777arc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,17 +60,24 @@ There are at least four ways you can use SigMF today, thanks to the community-su
 
 ## Contributing
 
-The SigMF standards effort is organized entirely within this Github repository.
+The SigMF standards effort is organized within this Github repository.
```
Comment (jacobagilbert) [inline README.md:63]: I would prefer if we qualify this with `core SigMF standards effort` as opposed to removing the specific language here.

> The core SigMF standards effort is organized entirely within this Github repository.

File: README.md [hunk 2]
```diff
@@ -60,17 +60,24 @@ There are at least four ways you can use SigMF today, thanks to the community-su
...
+a new issue and provide your input, even if it's not a traditional issue.
+Changes to the specification only occur through [Pull Requests](https://github.com/gnuradio/SigMF/pulls).
 
 This ensures that the history and background of all discussions and changes are
 maintained for posterity.
 
 Anyone is welcome to get involved - indeed, the more people involved in the
 discussions, the more useful the standard is likely to be.
 
+## Extensions
```
Comment (jacobagilbert) [inline README.md:75]: Suggest:

> The "Core" SigMF standard is intentionally kept limited in scope, additional metadata fields can be added through [SigMF Extensions](https://github.com/gnuradio/SigMF/blob/sigmf-v1.x/sigmf-spec.md#extension-namespaces). For example, the [signal extension](https://github.com/gnuradio/SigMF/blob/sigmf-v1.x/extensions/signal.sigmf-ext.md) provides a standard way to specify modulation schemes and other attributes of wireless comms signals. Several general purpose canonical extensions live within this repository directly in the [extensions directory](https://github.com/gnuradio/SigMF/tree/sigmf-v1.x/extensions), while others are maintained by third parties. Below we include a listing of some popular, compliant SigMF extensions. To have your extension reviewed for included on this list, please open a PR adding the repository to the list below:

**Review by jacobagilbert — CHANGES_REQUESTED** [review]

**Comment by jacobagilbert** [comment]
@777arc I added some language tweaks here. Can you also split up line 75 consistent with the other document formatting?

File: README.md (hunk 1 above)
Comment (bhilburn) [reply README.md:63]: I recommend using a word other than "core" since we tend to overload that term in the SigMF context, but agreed.

**Review by bhilburn — APPROVED** [review]

**Pull request merged**